7.1.0 Build Errors

#### Expected behaviour
Application Compiles

#### Actual behaviour
Application Doesn't Compile

#### Live demo with steps to reproduce
Build new application forcing Highcharts 7.1.0 (don't have a live link) to build in production mode.
```
Step 10/15 : RUN ng build -c $env
 ---> Running in 4d2c35869724

Date: 2019-04-01T18:52:05.492Z
Hash: e2f9d13207ff3a740b4a
Time: 24979ms
chunk {main} main.df7d516da5b815bde751.js (main) 563 bytes [initial] [rendered]
chunk {polyfills} polyfills.aa616c185dbdafc86f55.js (polyfills) 613 bytes [initial] [rendered]
chunk {runtime} runtime.618bcded6741c4c27fdf.js (runtime) 6.04 kB [entry] [rendered]
chunk {scripts} scripts.3482e92cd19a7658a278.js (scripts) 120 kB [entry] [rendered]
chunk {styles} styles.99f07f9c34b4593ac27a.css (styles) 488 kB [initial] [rendered]
[91m
ERROR in node_modules/highcharts/highcharts.d.ts(128972,17): error TS2304: Cannot find name 'PlotNetworkDataLabelsFormatterCallbackFunction'.
node_modules/highcharts/highcharts.d.ts(128982,21): error TS2304: Cannot find name 'PlotNetworkDataLabelsFormatterCallbackFunction'.
node_modules/highcharts/highcharts.d.ts(128987,19): error TS2304: Cannot find name 'PlotNetworkDataLabelsTextPath'.
node_modules/highcharts/highcharts.d.ts(128992,15): error TS2304: Cannot find name 'PlotNetworkDataLabelsTextPath'.
node_modules/highcharts/highcharts.d.ts(136653,17): error TS2304: Cannot find name 'PlotPackedBubbleDataLabelsFormatterCallbackFunction'.
node_modules/highcharts/highcharts.d.ts(136658,15): error TS2304: Cannot find name 'PlotPackedBubbleDataLabelsTextPath'.
node_modules/highcharts/highcharts.d.ts(237720,14): error TS2304: Cannot find name 'SeriesOrganizationNodesLayoutValues'.
```

#### Product version
<!--- Highcharts, Highstock or Highmaps plus version number -->
Highcharts 7.1.0
7.0.3 builds just fine.

--- a/js/modules/organization.src.js
+++ b/js/modules/organization.src.js
@@ -11,7 +11,7 @@
  * node's children will hang below their parent, allowing a tighter packing of
  * nodes in the diagram.
  *
- * @typedef {"normal"|"hanging"} Highcharts.SeriesOrganizationNodesLayoutValues
+ * @typedef {"normal"|"hanging"} Highcharts.SeriesOrganizationNodesLayoutValue
  */
 
 'use strict';
@@ -503,7 +503,7 @@ H.seriesType(
  * @sample highcharts/demo/organization-chart
  *         Hanging layout
  *
- * @type      {Highcharts.SeriesOrganizationNodesLayoutValues}
+ * @type      {Highcharts.SeriesOrganizationNodesLayoutValue}
  * @default   normal
  * @apioption series.organization.nodes.layout
  */

--- a/test/typescript-lint/index.d.ts
+++ b/test/typescript-lint/index.d.ts
@@ -1,2 +1,14 @@
-// required file for DTSLint
+// Lint type tests
 import * as Highcharts from 'highcharts';
+import MfiIndicator from 'highcharts/indicators/mfi';
+
+MfiIndicator(Highcharts);
+
+test_PlotMfiParamsOptions();
+
+function test_PlotMfiParamsOptions() {
+    // $ExpectError
+    const options: Highcharts.PlotMfiParamsOptions = {
+        index: 123
+    };
+}

--- a/test/typescript-lint/tsconfig.json
+++ b/test/typescript-lint/tsconfig.json
@@ -54,6 +54,9 @@
         "../../code/modules/data.d.ts",
         "../../code/modules/data.src.d.ts",
 
+        "../../code/modules/dotplot.d.ts",
+        "../../code/modules/dotplot.src.d.ts",
+
         "../../code/modules/drag-panes.d.ts",
         "../../code/modules/drag-panes.src.d.ts",
 

--- a/test/typescript/indicators/mfi.ts
+++ b/test/typescript/indicators/mfi.ts
@@ -2,12 +2,3 @@ import * as Highcharts from 'highcharts';
 import MfiIndicator from 'highcharts/indicators/mfi';
 
 MfiIndicator(Highcharts);
-
-test_PlotMfiParamsOptions();
-
-function test_PlotMfiParamsOptions() {
-    // $ExpectError
-    const options: Highcharts.PlotMfiParamsOptions = {
-        index: 123
-    };
-}

--- a/test/typescript/modules/annotations.ts
+++ b/test/typescript/modules/annotations.ts
@@ -1,7 +1,13 @@
 import * as Highcharts from 'highcharts';
+/*
+// @todo annotations module failes because of type mismatches
+
 import AnnotationsModule from "highcharts/modules/annotations";
 
 AnnotationsModule(Highcharts);
+
+test_basic();
+test_advanced();
 
 function test_basic() {
     Highcharts.chart('container', {
@@ -210,3 +216,4 @@ function test_advanced() {
         }]
     });
 }
+*/

--- a/test/typescript/modules/heatmap.ts
+++ b/test/typescript/modules/heatmap.ts
@@ -3,17 +3,19 @@ import HeatmapModule from 'highcharts/modules/heatmap';
 
 HeatmapModule(Highcharts);
 
+test();
+
 function test() {
     const colors = Highcharts.getOptions().colors;
+    const maxSafeInteger = 9007199254740991;
     const tooltipFormatter = function(
         this: Highcharts.TooltipFormatterContextObject
     ) {
-        const point = this.point as any; // @todo make Point.value public
         const series = this.series as any; // @todo make Axis.categories public
         return (
             `<b>${series.xAxis.categories[this.point.x]}</b> sold <br>
-            <b>${point.value}</b> items on <br>
-            <b>${series.yAxis.categories[this.point.y || Number.MAX_SAFE_INTEGER]}</b>`
+            <b>${this.point.value}</b> items on <br>
+            <b>${series.yAxis.categories[this.point.y || maxSafeInteger]}</b>`
         );
     };
     Highcharts.chart('container', {

